addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/SystemControlXML.java

```diff
@@ -32,23 +37,93 @@
  *
  * @author David Gräff - Initial contribution
  * @author Tomasz Maruszak - refactoring, HTR-xxxx Zone_2 compatibility
- *
  */
 public class SystemControlXML implements SystemControl {
+
+    private static final Set<String> MODELS_WITH_PARTY_SUPPORT = new HashSet<>(Arrays.asList(RX_A2000));
+
+    protected Logger logger = LoggerFactory.getLogger(SystemControlXML.class);
```

Comment:
Can it be private (and final), I prefer every class to have its own logger for clarity